db/patch-sub-users.sql

```diff
@@ -0,0 +1,10 @@
+-- You should not modify if this have pushed to Github, unless it does serious wrong with the db.
+BEGIN TRANSACTION;
+
+CREATE TABLE [sub_users](
```

Comment:
No idea why commits present here, but the `File changed` still shows nothing.

However, creating a new table `sub_users` which is not what I want to do unfortunately. As in this way, I believe all user related features have to re-implemented in order to handle both `user` and `sub_users`, which is not ideal. The best way should be keep using `user` table only.

As I said before, there will be so many discussions on how to design in code or in UI/UX, so I do not want anyone working on it.

If you want to contribute to this project, I would recommend that you should start from other smaller feature first.

Reply:
Yes, the users table is already set up for multiple users as each monitor has a foreign key referencing the user that created it. The only changes I think that need to be made to the actual database would be some sort of field for access level in the users table and possibly a visibility field in monitors/status pages. A group system might be something to look into to make management easier. @louislam, is there a discussion/issue that could be used for people to discuss the implementation of multi users to help keep everything in one place.

Reply:
I mainly made that for testing because I wasn’t sure if the user part was for multiple or not, definitely should of asked about that, but I could easily just turn that into a permission implementation.